src/browser/api_protocol/api_handlers/authorization.js

```diff
@@ -99,7 +99,8 @@ function onRequestAuthorization(id, data) {
     const uuid = data.payload.uuid;
     const authObj = pendingAuthentications.get(uuid);
     const externalConnObj = Object.assign({}, data.payload, {
-        id
+        id,
+        configUrl: authObj.authReqPayload && authObj.authReqPayload.configUrl
```

Comment:
This can maybe end up being `configUrl: null` or other falsy values. Will it brake somewhere down the line then? Maybe be should be this instead: 
```
const externalConnObj = Object.assign({}, data.payload, { id });

if (authObj.authReqPayload) {
   externalConnObj.configUrl = authObj.authReqPayload.configUrl;
}
```

Reply:
added check for authObj